calitp-data-analysis/tests/helpers.py

```diff
@@ -0,0 +1,20 @@
+from contextlib import contextmanager
+
+CI_SCHEMA_NAME = "calitp_py"
+
+
+@contextmanager
+def as_calitp_user(user):
+    import os
```

Comment:
Suggestion: add a quick docstring about why this exists so people don't think it should be used for other purposes